src/metabase/pulse/render.clj

```diff
@@ -736,11 +737,14 @@
                          (external-link-image-bundle render-type))]
       {:attachments (when image-bundle
                       (image-bundle->attachment image-bundle))
-       :content     [:table {:style (style {:margin-bottom :8px
-                                            :width         :100%})}
+       :content     [:table {:style (style {:margin-bottom   :8px
+                                            :border-collapse :collapse
+                                            :width           :100%})}
                      [:tbody
                       [:tr
-                       [:td [:span {:style (style header-style)}
+                       [:td {:style (style {:padding :0
```

Comment:
Not a big deal, and the end result is the same, but in other areas of this file when we have an integer as a value for a style it's just `0` instead of `:0`.